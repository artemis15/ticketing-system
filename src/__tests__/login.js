import React from 'react';
import { shallow } from 'enzyme';
import Login from '../components/login';

describe('Login Render', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<Login />);

        expect(component).toMatchSnapshot();
    });
});
