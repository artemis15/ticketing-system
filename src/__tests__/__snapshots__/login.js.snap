// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Render should render correctly in "debug" mode 1`] = `ShallowWrapper {}`;
